build.gradle

```diff
@@ -123,7 +123,7 @@ check.dependsOn installDist
 
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
-    compile 'org.broadinstitute:gatk:4.alpha.1-129-gef0152f-20160620.203528-1'
+    compile 'org.broadinstitute:gatk:4.alpha.1-146-gfacef8d-SNAPSHOT'
```

Comment:
Remember to include the full timestamp of the desired snapshot here to make the dependency less brittle (otherwise the newest snapshot with this version will be used).